combat-trainer.lic

```diff
@@ -2153,7 +2161,9 @@ class GameState
   end
 
   def melee_attack_verb
-    if attack_override
+    if @use_tactics_combos && !offhand? && brawl_tactics? && !backstab? && tactics_analyze?
+      @tactics_combo_array.shift
+    elsif attack_override
       attack_override
```

Comment:
I would also expect barb combos to be a higher priority than general combos. What do you think?